analyzers/src/SonarAnalyzer.VisualBasic/Facade/VisualBasicSyntaxKindFacade.cs

```diff
@@ -27,6 +27,7 @@ internal sealed class VisualBasicSyntaxKindFacade : ISyntaxKindFacade<SyntaxKind
         public SyntaxKind InvocationExpression => SyntaxKind.InvocationExpression;
         public SyntaxKind[] ObjectCreationExpressions => new[] {SyntaxKind.ObjectCreationExpression};
         public SyntaxKind[] ClassAndRecordDeclaration => new[] {SyntaxKind.ClassBlock};
+        public SyntaxKind[] TypeDeclaration => ClassAndRecordDeclaration;
```

Comment:
why not `StructureBlock` here? And also `InterfaceBlock` and `EnumBlock` as it's a generic `TypeDeclaration`

see [on sharplab](https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBsQDdhJiaMkAmIA1AD4CSYADgPYIxQAEAygJ6xxgCwAUAAoBXYCgCWAYxYwEAsTAEI4LXgFEAdgUnTZ8xbxWqBYRssa9G5xgHlVcfRrWG9PcqvgIAZhDGLydxi7dPbyA==)